init-tools.sh

```diff
@@ -109,9 +109,10 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
 
     Tools/crossgen.sh $__scriptpath/Tools
 
+    mkdir -p "$(dirname "$__INIT_TOOLS_DONE_MARKER")"
```

Comment:
@janvorli Could you let me know who creates this directory? I checked again, but this directory is still missing:
```
Crossgen finished.
touch: cannot touch ‘/home2/parjong/projects/dotnet/coreclr/Tools/2.0.0-prerelease-01528-03/done’: No such file or directory
Done initializing tools.
```